lib/selection/service/flag_applications.rb

```diff
@@ -0,0 +1,65 @@
+require 'selection/distance'
+
+module Selection
+  module Service
+    class FlagApplications
+      FLAGS           = %w(remote_team male_gender age_below_18 less_than_two_coaches)
+      STUDENT0_COORDS = %w(student0_application_location_lat student0_application_location_lng)
+      STUDENT1_COORDS = %w(student1_application_location_lat student1_application_location_lng)
+      CITY_THRESH     = 100
+      GENDER_FIELDS   = %w(student0_application_gender_identification student1_application_gender_identification)
+      MALE_MATCHER    = ->(a) { a =~ /\A\s*male\z|\A\s*man\z|\A\s*men\z|\A\s*guy\z/i }
```

Comment:
Yes, could be - looks way cleaner actually 🎉 
I'm not a regex-ninja, just quickly hacked this one together...